.github/workflows/ci.yml

```
@@ -81,6 +81,23 @@ jobs:
       - name: Test import
         run: |
           python -c "import ansys.tools.report as report; report.version();"
+          echo "::set-output name=PYANSYS_TOOLS_REPORT_VERSION::$(python -c "import ansys.tools.report as report; report.version()")"
+
+      - name: Generate wheelhouse
+        run: pip wheel . -w wheelhouse
+
+      - name: Zip wheelhouse
+        uses: vimtor/action-zip@v1
+        with:
+          files: wheelhouse
+          dest: ${{ env.PACKAGE_NAME }}-v${{ steps.version.outputs.PYANSYS_TOOLS_REPORT_VERSION }}-wheelhouse-${{ runner.os }}-${{ matrix.python-version }}.zip
+
+      - name: Upload Wheelhouse
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ env.PACKAGE_NAME }}-v${{ steps.version.outputs.PYANSYS_TOOLS_REPORT_VERSION }}-wheelhouse-${{ runner.os }}-${{ matrix.python-version }}
+          path: '*.zip'
+          retention-days: 7
 
   build_test:
     name: Build and Unit Testing
```

Suggesting these lines @RobPasMue:

```suggestion
  docs:
    name: Build and deploy documentation
```

---

Anyway, maybe this fix could be taken into account later on. I'll keep it in mind. Thanks @jorgepiloto.